ens/main.py

```diff
@@ -30,6 +31,27 @@
 )
 
 ENS_MAINNET_ADDR = '0x314159265dD8dbb310642f98f50C066173C1259b'
+ENS_ROPSTEN_ADDR = '0x112234455C3a32FD11230C42E7Bccd4A84e02010'
+ENS_RINKEBY_ADDR = '0xe7410170f87102DF0055eB195163A03B7F2Bff4A'
+ENS_GOERLI_ADDR = '0x112234455C3a32FD11230C42E7Bccd4A84e02010'
+
+
+def get_address_for_network(net):
```

Comment:
Could this helper function and the 4 top-level constants just be replaced with a simple `dict`?

For example:
```python
NET_VERSION_TO_ENS_ADDR = {
    1: <MAINNET ADDR>,  # Mainnet
    2: <ROPSTEN ADDR>,  # Ropsten
    ....
}

get_address_for_network = NET_VERSION_TO_ENS_ADDR.__getitem__
```
I think `get_address_for_network` would just be replaced with `NET_VERSION_TO_ENS_ADDR[id]` instead of creating that reference to `NET_VERSION_TO_ENS_ADDR.__getitem__`.  I just included that new definition to illustrate what I'm getting at.

Reply:
@davesque ok, changed!